lib/afterpay.rb

```diff
@@ -8,6 +8,7 @@
 require 'afterpay/api/payment'
 require 'afterpay/api/payment/capture'
 require 'afterpay/api/payment/refund'
+require 'afterpay/api/payment/void'
```

Comment:
you should move this `require` in the previous commit

Reply:
Done!